apps/guardian-ui/src/components/ConsensusInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Tooltip } from '@chakra-ui/react';
+import { InfoOutlineIcon } from '@chakra-ui/icons';
+
+export const ConsensusInfo = () => {
+  const totalGuardians = 10;
```

Comment:
Can we make the example dynamic, based on the current number of guardians selected in the input field?